augur/utils.py

```diff
@@ -544,21 +546,30 @@ def run_shell_command(cmd, raise_errors = False, extra_env = None):
     overlayed onto the default subprocess environment.
     """
     env = os.environ.copy()
+    out = ''
 
     if extra_env:
         env.update(extra_env)
 
     try:
+        shargs = ['-c', "set -euo pipefail; " + cmd]
+
+        if os.name == 'posix':
+            shellexec = ['/bin/bash']
+        else:
+            # We try best effort on other systems. For now that means nt/java.
+            shellexec = ['env', 'bash']
+
         # Use check_call() instead of run() since the latter was added only in Python 3.5.
-        subprocess.check_call(
-            "set -euo pipefail; " + cmd,
-            shell = True,
-            executable = "/bin/bash",
-            env = env)
+        out = subprocess.check_output(
+            shellexec + shargs,
+            shell = False,
+            stderr=subprocess.STDOUT)
 
     except subprocess.CalledProcessError as error:
         print_error(
-            "shell exited {rc} when running: {cmd}{extra}",
+            "{out}\nshell exited {rc} when running: {cmd}{extra}",
+            out = out,
```

Comment:
Based on the doc for `check_output`, it seems like `out = out` needs to be `error.output`? https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError.output

Reply:
```suggestion
            out = error.output,
```